scheme/src/EntityGeneratorBundle/Tools/InterfaceGenerator.php

```diff
@@ -41,7 +41,11 @@ public function createEmptyInterfaces()
      */
     public function writeEntityClass(ClassMetadataInfo $metadata, $outputDirectory)
     {
-        return parent::writeEntityClass($this->transformMetadata($metadata), $outputDirectory);
+        try {
+            return parent::writeEntityClass($this->transformMetadata($metadata), $outputDirectory);
+        } catch (\Exception $e) {
+            throw $e;
+        }
     }
```

Comment:
Unneeded reimplementation of writeEntityClass?

Reply:
Fixed, thanks :)